modal/client.py

```diff
@@ -210,6 +210,8 @@ async def from_env(cls, _override_config=None) -> "_Client":
             token_id = c["token_id"]
             token_secret = c["token_secret"]
             if task_secret:
+                if token_id or token_secret:
+                    warnings.warn("Modal token id and secret are ignored inside containers")
```

Comment:
Should we clarify that these are the one in environment variables?

Reply:
We currently don't read `.modal.toml` in the container: https://github.com/modal-labs/modal-client/blob/f804d0278fac7d3399bef8f3e87529ec1301488a/modal/config.py#L106-L124

Although I guess that's a different way that we might confusingly ignore tokens!

Reply:
Nice, maybe I overreacted updating the warning message then! It's probably fine to err on the side of being extra verbose